<launch>
    
    <rosparam file="$(find amr_trajectory_viz)/params/logger.yaml" />
   
    <node name="logger_node" pkg="amr_trajectory_viz" type="logger_node" output="screen" />
        
</launch>